src/schema/rules/files/raw/audio.yaml

```diff
@@ -0,0 +1,23 @@
+---
+audio:
+  datatypes:
+    - stimuli
+  suffixes:
+    - audio
+  extensions:
+    - .wav
+    - .WAV
```

Comment:
I do not think we should allow/encourage mixed casing allowed!  Let's stick to lower case . we have .json and .tsv and no .JSON and .TSV . I do not see why these files need to be different